doc/material/index.rst

```diff
@@ -4,8 +4,9 @@ MESSAGEix-Materials
 Description
 ===========
 
-This module adds material stock and flow accounting in MESSAGEix-GLOBIOM. The implementation currently includes four key energy/emission-intensive
-material industries: Iron&Steel, Aluminum, Cement, and Chemicals.
+This module adds material stock and flow accounting in MESSAGEix-GLOBIOM. The
```

Comment:
Per the [code style](https://docs.messageix.org/en/latest/contributing.html#documentation), please:
> start each sentence on a new line, and do not hard-wrap within sentences.